### PR TITLE
add a human-readable alias for Korean

### PR DESCRIPTION
**Pull request by burner1024**
Hi, this is a human-readable alias for Korean language

### CODE REVIEW

File: weblate/lang/data.py [hunk 1]
```diff
@@ -346,6 +346,7 @@
     'schinese': 'zh_Hans',
     'chinese_zh': 'zh_Hant',
     'tchinese': 'zh_Hant',
+    'korean': 'ko_KR',
```
Comment (nijel) [inline weblate/lang/data.py]: Wouldn't it be better to alias it to `ko` only? The country specification is usually not needed and the alias should be consistent with others which do not use country unless necessary...

Comment (burner1024) [reply weblate/lang/data.py]: Will it work like that? I see no definition for "ko" in data.py

Comment (nijel) [reply weblate/lang/data.py]: The language definitions have been recently moved to separate file:

https://github.com/WeblateOrg/weblate/blob/4ded14d535b02c306f1cfaa5268557ef4711e3e5/weblate/langdata/languages.py#L943-L948

Comment (burner1024) [reply weblate/lang/data.py]: changed to ko

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**